modules/support-gather-data.adoc

```diff
@@ -31,7 +31,7 @@ If this command fails, for example if you cannot schedule a Pod on your cluster,
 +
 [NOTE]
 ====
-If your cluster is using a restricted network you must import the default must-gather image before running the `oc adm must-gather` command.
+If your cluster is using a restricted network and the mirror registry have a trusted CA, you should add it through xref:../openshift_images/image-configuration.adoc#images-configuration-cas_image-configuration[Configuring additional trust stores]. And then you must import the default must-gather image as ImageStream before running the `oc adm must-gather` command. 
```

Comment:
@bysnupy, you can't put an xref in a module. I think the note here should be:

```suggestion
If your cluster is using a restricted network you must take additional steps. If you mirror registry has a trusted CA, you must first add the trusted CA to the cluster. For all clusters on restricted networks, you must import the default `must-gather` image as ImageStream before using the `oc adm must-gather` command. 
```

If this is something that users need to do anyway, I think that we should tell them somewhere in the installation process.

Is it possible for users to make the changes that they need to around here in the process? https://docs.openshift.com/container-platform/4.5/installing/install_config/installing-restricted-networks-preparations.html#installation-preparing-restricted-cluster-to-gather-support-data_installing-restricted-networks-preparations
If so, I think it would be better to add more info there. 

If not, I think that adding the link to the process for adding the mirror registry CA to the OCP cluster should be in the "next steps" section of the restricted network installations, like

```
* If the mirror registry that you used to install your cluster has a trusted CA, add it to the cluster by xref:../openshift_images/image-configuration.adoc#images-configuration-cas_image-configuration[Configuring additional trust stores].
```

What do you think?

Reply:
@kalexand-rh Your point is correct, and installation section is an ideal place to provide the requirement for using ImageStream on the restricted network environment. But unfortunately the must-gather image import procedures are shown at multiple sections. It leads users confusion. I thinks we should integrate the must-gather image import description at one place, if we can not do that, we should provide the same thing a about requirement of the trusted CA on all the related sections. How about your thought ?

Reply:
@bysnupy, thank you for updating this note!

I think that it's ok to add the requirement about the trusted CA to each of the restricted network installation topics. Do you want me to modify your PR to do that?

Reply:
@kalexand-rh Yes, I do. Thank you for your help !